src/SFA.DAS.FindEmploymentSchemes.UnitTests/Web/Models/SchemeTests.cs

```diff
@@ -38,7 +38,12 @@ private static Scheme TestScheme(string url)
                     "pay--minimum-wage", "motivations--full-time-role", "motivations--diversity-or-responsibility",
                     "scheme-length--4-months-to-12-months", "scheme-length--a-year-or-more",
                 },
-                null,
+                new CaseStudy[]
+                {
+                    new CaseStudy("name", "title", new HtmlString(@"<p>Case study content here</p>"))
```

Comment:
Before we can merge this into the release branch, we'll either have to have the real case study content in, or remove the test content.

Reply:
We could delete the test content and regenerate after it passes testing, if we do it sooner there'll be no content to test with

Reply:
Removed case study test data